Chummer/data/gear.xml

```diff
@@ -1532,6 +1532,68 @@
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
     </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
```

Comment:
While Chummer can (now) handle an availability entry of `-`, it's better to use `0` so that its value can be processed as a number for things like the amend system. This applies to all of the other slingshot ammo you added, though I will only make this comment for this entry.

Reply:
Changed to 0 for all 4 of them.